jep/1/README.adoc

```diff
@@ -0,0 +1,939 @@
+changes as evidenced by many of the Jenkins 2.0 mailing list threads
+footnote:[https://groups.google.com/d/msg/jenkinsci-dev/vbXK7JJekFw/BlEvO0UxBgAJ],
+which ballooned into threads with 100+ replies and sufficient chaos to be very
+difficult for those who weren't full-time Jenkins developers to understand.
+
+
+The Jenkins Enhancement Proposal aims to address both of these major issues by
+providing an understood process for making sizable, but understandable,
+enhancements to Jenkins.
+
+=== Benefits to existing developers
+
+JEP provides a systematic approach for vetting and developing new proposals and
+ideas for Jenkins. By encouraging "everybody to follow the rules" it will be
+easier for existing developers to get their ideas and changes into Jenkins
+without finding themselves mired in unspoken cultural norms within the project.
+
+=== Benefits to future developers
+
+By providing clear, understandable, and bite-sized design documents which would
+explain various subsections of Jenkins. JEPs also make it clearer how an
+ambitious new developer to the Jenkins project can propose, and make progress
+upon, a new idea they have for Jenkins.
+
+Overall, less chaos and more productivity is the rationale for JEP.
+
+== Reasoning
+
+The Python community, whose process JEP is modeled after, have successfully
+navigated several large-scale reworkings of Python and it's related tools and
+processes over the past decade. This includes most notably the multi-year
+project of Python 3 (formerly Python 3000).
+
+Their Python Enhancement Proposals are largely consensus driven, which is
+_mostly_ how work is done presently in the Jenkins project,
+footnote:[https://groups.google.com/d/msgid/jenkinsci-dev/824CAC89-7A49-478A-9904-5C77D8FF5A80%40beckweb.net]
+footnote:[https://groups.google.com/d/msgid/jenkinsci-dev/CAPbPdObKcXxZ2rgGdx6Z2HVKwH9mE_gkVbB1GOeCEhmZ7JkfwQ%40mail.gmail.com]
+footnote:[https://groups.google.com/d/msgid/jenkinsci-dev/CA%2BnPnMz-m49TK7Em%2BxBNb%2BV98dBCz9CrrPXg3uW6%2B_x3KX5gOQ%40mail.gmail.com]
+ making the PEP model relatively straightforward to graft onto our existing
+ processes for making proposals and deciding upon changes.
+
+=== Differences from Python PEP
+
+The process by which a number of link:https://apache.org[Apache] projects are
+operated was also considered, but the Python Enhancement Proposal process was
+by far the most well-documented and obviously successful approach considered to
+project improvement (technical and otherwise) over time.
+
+The Python process uses "Rationale" as the heading for the section for describing design decisions.
+However the meaning of "Rationale" is similar to "Motivation" in some contexts.
+We decided to use "Reasoning" instead to avoid confusion.
+
+=== Requirements-level terms
+
+Some non-native english speakers commented on the
+<<requirement-levels, Must/Should/May>> mentioned that "should" is a synonym of "must",
```

Comment:
Sure, *should* is used the same way your boss *asks* you to *please* do something, but it's understood to not be a request that is easily refused. Just the way some people communicate.

Reply:
@daniel-beck I'm not sure what action you'd like taken here.

Reply:
Unless we're trying to address every review comment in its own section in the doc, I don't think it's useful to have this particular section. The original comment that spawned this directly contradicted dictionaries.

Reply:
@daniel-beck
Are you saying you object to "this section" as in this part the "Reasoning" isn't needed? 
Or are you saying the "Must/Should/May" section isn't needed?